common/src/main/java/com/linecorp/centraldogma/internal/ApplicationToken.java

```diff
@@ -0,0 +1,11 @@
+package com.linecorp.centraldogma.internal;
+
+/**
+ * Holds a default application token.
+ */
+public class ApplicationToken {
```

Comment:
@minwoox, thank you so much for the detailed explanation. It helped me a lot 👍

It seems like com.linecorp.centraldogma.client.armeria.AbstractCentralDogmaBuilder sets com.linecorp.centraldogma.internal.CsrfToken#ANONYMOUS as accessToken by default. So the question is, should we use CsrfToken#ANONYMOUS when checking anonymous token, or should we create a new ApplicationToken#ANONYMOUS for the purpose of checking anonymous token?

Reply:
I've added a commit that uses `CsrfToken#ANONYMOUS` and remove `ApplicationToken`